src/include/optimizer/stats/cost.h

```diff
@@ -137,6 +145,80 @@ class Cost {
   inline static double InnerHashJoin() { return DEFAULT_COST; }
 
   /*
+   * Join with Sampling
+   */
+  static double InnerNLJoinWithSampling(
+      const std::shared_ptr<TableStats>& left_input_stats,
+      const std::shared_ptr<TableStats>& right_input_stats,
+      std::shared_ptr<TableStats>& output_stats,
+      const std::shared_ptr<expression::AbstractExpression> predicate);
```

Comment:
What if there are multiple join predicates?

Reply:
Multiple joins can also be represented in the format of AbstractExpression connected with conjunction expressions. So there'll be only one join predicate.